src/core/public/doc_links/doc_links_service.ts

```diff
@@ -77,10 +80,64 @@ export class DocLinksService {
           auditdModule: `${ELASTIC_WEBSITE_URL}guide/en/beats/auditbeat/${DOC_LINK_VERSION}/auditbeat-module-auditd.html`,
           systemModule: `${ELASTIC_WEBSITE_URL}guide/en/beats/auditbeat/${DOC_LINK_VERSION}/auditbeat-module-system.html`,
         },
+        appSearch: {
+          apiRef: `${ELASTIC_WEBSITE_URL}guide/en/app-search/${DOC_LINK_VERSION}/api-reference.html`,
```

Comment:
This could be DRY'd quite a bit by having a const for `APP_SEARCH_BASE` or something similar on line 25 above:
```JavaScript
const APP_SEARCH_BASE = `${ELASTIC_WEBSITE_URL}guide/en/app-search/${DOC_LINK_VERSION}`
```
and then for these:
```JavaScript
apiRef: `${APP_SEARCH_BASE}/api-reference.html`,
```
Same goes for Enterprise Search and Workplace Search

Reply:
Yes, we've done that for some other books, so it's feasible. We've talked about removing those, however, since they enable folks to create hard-coded links elsewhere in the code by using that base. It also requires some changes to the doc link checker, but I can set that up.